products/cakephp.md

```diff
@@ -232,11 +230,26 @@ releases:
 
 ---
 
-> [CakePHP](https://cakephp.org/) is a free & open source PHP web development framework.  It follows
-> the model–view–controller (MVC) approach and is written in PHP, modeled after the concepts of Ruby
-> on Rails, and distributed under the [MIT License](https://en.wikipedia.org/wiki/MIT_License).
+> [CakePHP](https://cakephp.org/) is a free and open source PHP web development framework. It
+> follows the model–view–controller (MVC) approach and is written in PHP, modeled after the concepts
+> of Ruby on Rails.
 
-Have a look at the [Bakery](https://bakery.cakephp.org/) for fresh [releases](https://bakery.cakephp.org/categories/release.html).
+CakePHP follows [Semantic Versioning](https://book.cakephp.org/4/en/release-policy.html). And, from
+3.x onward, major versions also have a name:
+
+| Version | Name       |
+|---------|------------|
+| 3.x     | Red Velvet |
+| 4.x     | Strawberry |
```

Comment:
The nick name of CakePHP 5.x is missing.

Reply:
Information that relates to version that are not yet released are never documented on endoflife.date.

Reply:
There is a beta:

https://github.com/cakephp/cakephp/releases/tag/5.0.0-beta1
https://bakery.cakephp.org/2022/12/19/cakephp_500beta1_released.html

Reply:
The endoflife.date project does not document releases that are not considered "stable" (such as RC/Alpha/Beta/Nightly). CakePHP 5.x name will have to be added when the 5.0.0 will be GA. I left a comment next to the `releases` key to remind it.